libra_swarm/src/main.rs

```diff
@@ -34,6 +35,9 @@ struct Args {
 fn main() {
     let args = Args::from_args();
     let num_nodes = args.num_nodes.unwrap_or(1);
+    // topology indicates the shape of the validator network
+    // ['num_nodes', 'num_full_nodes', 'num_full_node_children']
```

Comment:
nitpick: no need to expose internal details in callsite
so instead maybe: 
    // topology indicates structure of the validator network
   // e.g. num of validators, num of full nodes and their children